src/spiders/cisa_spider.py

```diff
@@ -0,0 +1,40 @@
+import scrapy
+
+from items import AlertItem
+
+
+class CisaSpider(scrapy.Spider):
+    """
+    Spider for the US-CERT website.
+    """
+
+    name = "US-CERT"
+    max_items = 10
+    start_urls = ["https://www.cisa.gov/uscert/ncas/current-activity"]
+    block_selector = "div.views-row"
+    link_selector = "descendant-or-self::h3/span/a/@href"
+    date_selector = (
+        "descendant-or-self::div[contains(@class,'entry-date')]/span[2]/text()"
+    )
+    title_selector = "descendant-or-self::h3/span/a/text()"
+    description_selector = "descendant-or-self::div[contains(@class,'field-content')]/p"
+
+    def parse(self, response):
```

Comment:
isn't this function getting repeated? if yes can we take it somewhere where it will be called by all spiders

Reply:
Yes, it is, but it is a method of the abstract class `Spider`, so I guess we can't take it anywhere. Please let me know if you can suggest an implementation. 
What I would do is add another layer of inheritance, abstract all the spiders implemented into one class, and have its arguments be the website URL and the selectors.